requirements/base.txt

```diff
@@ -408,7 +404,6 @@ cryptography==41.0.5 \
     --hash=sha256:ec3b055ff8f1dce8e6ef28f626e0972981475173d7973d63f271b29c8a2897da \
     --hash=sha256:fba1e91467c65fe64a82c689dc6cf58151158993b13eb7a7f3f4b7f395636723
     # via
-    #   auth0-python
```

Comment:
Odd.

Reply:
Yeah, I'm not sure what's up with this. The version of `auth0-python` we're using _certainly_ depends on it. It seems like it's some odd artifact of using `--no-upgrade`. If I re-run `pin.sh` without that, `auth0-python` is still listed here.